src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -225,6 +229,12 @@ public void Initialize(TestLoggerEvents events, Dictionary<string, string> param
                 bool.TryParse(noprogress, out DisableProgress);
             }
 
+            if (this.verbosityLevel == Verbosity.Quiet)
```

Comment:
>if (this.verbosityLevel == Verbosity.Quiet) [](start = 12, length = 43)

This is not needed. You can always get the targetframework despite verbosity being quite.

Reply:
we are only doing this in quiet case for now.